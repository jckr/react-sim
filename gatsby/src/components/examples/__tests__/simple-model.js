import React from 'react';
import renderer from 'react-test-renderer';
import { Frame } from '../simple-model';

describe('Simple Model', () => {
  it('model renders properly', () => {
    const tree = renderer.create(<Frame tick={10} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
