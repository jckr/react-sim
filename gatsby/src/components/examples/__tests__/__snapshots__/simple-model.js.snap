// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple Model model renders properly 1`] = `
<div
  className="css-l3rx45"
>
  <div
    className="css-14lu1a5"
  >
    <div
      className="css-15yps5k"
    />
    <div
      className="css-15yps5k"
    />
    <div
      className="css-15yps5k"
    />
    <div
      className="css-15yps5k"
    />
    <div
      className="css-15yps5k"
    />
    <div
      className="css-15yps5k"
    />
    <div
      className="css-15yps5k"
    />
    <div
      className="css-15yps5k"
    />
    <div
      className="css-15yps5k"
    />
    <div
      className="css-15yps5k"
    />
  </div>
  <div
    className="css-14lu1a5"
  >
    <div
      className="css-15yps5k"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
  </div>
  <div
    className="css-14lu1a5"
  >
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
  </div>
  <div
    className="css-14lu1a5"
  >
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
  </div>
  <div
    className="css-14lu1a5"
  >
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
  </div>
  <div
    className="css-14lu1a5"
  >
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
  </div>
  <div
    className="css-14lu1a5"
  >
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
  </div>
  <div
    className="css-14lu1a5"
  >
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
  </div>
  <div
    className="css-14lu1a5"
  >
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
  </div>
  <div
    className="css-14lu1a5"
  >
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
    <div
      className="css-1hzvbw9"
    />
  </div>
</div>
`;
